dtale/dash_application/utils.py

```diff
@@ -5,4 +5,4 @@ def get_data_id(pathname):
     """
     Parses data ID from query path (ex: 'foo/bar/1' => '1')
     """
-    return global_state.find_data_id(pathname.split("/")[-1])
```

Comment:
So I think you still want to keep this so that in case someone tries to hit the path `dtale/charts` with no data_id on the end of the path.  Otherwise this is what you end up seeing: 
![image](https://user-images.githubusercontent.com/11547371/108017952-4ffadf80-6fe4-11eb-840d-ad68e67c05f0.png)

Reply:
What should be the default behaviour of dtale/charts without data_id?

Reply:
modified to mimic original behaviour.